bin/forklift

```diff
@@ -0,0 +1,273 @@
+#!/usr/bin/env python3
+
+# Needs at least python 3.4 or the pathlib library
```

Comment:
No, I think it's in EPEL7 and there's an SCL. I'm trying to focus on Python 3 but this script can easily be rewritten to support both python 2 and python 3.